docs/contributing/workflow.rst

```diff
@@ -36,6 +38,33 @@ Create a new branch
 #. Navigate to the :file:`PlasmaPy/` directory that contains the clone
    of your repository.
 
+#. In the terminal, run:
+
+   .. code-block::
+
+      git status
+
+   If the output ends with ``nothing to commit, working tree clean``,
+   then proceed to the next step. Otherwise, click to open the next
+   section.
+
+   .. collapse:: Instructions if there are uncommitted changes
```

Comment:
The [`collapse`](https://sphinx-toolbox.readthedocs.io/en/stable/extensions/collapse.html) directive from [`sphinx-toolbox`](https://sphinx-toolbox.readthedocs.io) lets us create a collapsible section to a page which is not open by default.